pmm-app/src/shared/components/Form/FieldAdapters/FieldAdapters.tsx

```diff
@@ -89,3 +90,29 @@ export const SelectFieldAdapter = ({
     </Field>
   );
 };
+
+export const RadioButtonGroupAdapter = ({
+  input, options, selected, label, meta, dataQa, ...props
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <Field label={label}>
+      <>
+        <RadioButtonGroup
+          {...input}
+          {...props}
+          options={options}
+          value={input.value || selected}
+          dataQa={dataQa}
+        />
+        {meta.touched && meta.error && (
```

Comment:
Should we reserve this space for a potential error message to avoid pushing content down when there is a validation error? I guess we shouldn't if this is consistent across the board, but let me ask anyway.

Reply:
Yes we should reserve the space like we are doing everywhere. Fixed it.